migrations/0002_analysis_user_extent.py

```diff
@@ -1,19 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Do not delete this file, it is a record of database structure version. Always change the new one you want to add --> 003. Deleting migration file only happens when the migration file is squashed.